lib/datadog/appsec/contrib/rack/request_middleware.rb

```diff
@@ -33,6 +33,21 @@ def call(env)
             ready = false
             context = nil
 
+            # If a customer has mounted a nested Rack app within their main app
+            # and instrumented boths apps:
```

Comment:
We generally don't refer to "customers" in code:

```suggestion
            # If a user has mounted a nested Rack app within their main app
            # and instrumented boths apps:
```

In fact we can remove the reference altogether with a passive tone:

```suggestion
            # If a nested Rack app is mounted within a main app
            # and both are instrumented:
```